### ISSUE TITLE
Reading ACL groups? (client/bot)

### ISSUE THREAD

**Issue by CaptainZidgel** [bkacjios/Lumble#5]
I do not know when the hook OnACL is sent, but it appears from the code that when it is sent it doesn't send [any information at all](https://github.com/bkacjios/Lumble/blob/master/modules/lumble/client/init.lua#L822). The information I am particularly interested in from the proto is [Mumble.ACL.ChanGroup.add ](https://github.com/mumble-voip/mumble/blob/master/src/Mumble.proto#L328)(the list of all users in a specific ACL group).
Is this information even available to the client? What would I need to learn in order to write this feature?
I decided to take a look at this as an alternative to lua-mumble since I know lua and could possibly modify the library when it was missing features (which I can't do with C) but it looks like I'm lost here too :p

**Comment by bkacjios** [comment]
I never really looked into the ACL stuff since I never really had a need for anything it provides. I don't know what it sends or when or how it all works.

**Comment by CaptainZidgel** [comment]
I made an issue on the Mumble github repo and found out how to get ACL information sent to the client.
I created a PR (#6) if you're interested.
The code doesn't exactly match the style of the rest but it works. Only thing I think could use some work is the event formatting. I tried to make it a little more friendly for an end user if they needed the information that I needed but ultimately the ACL event isn't as friendly as the events normally are in, say, lua-mumble (I don't know how friendly the events are in this library as I haven't started porting my bot over yet).
